tasks/files-to-granules/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@cumulus/files-to-granules",
+  "version": "1.1.3-alpha1",
+  "description": "Converts array-of-files input into a granules object by extracting granuleId from filename",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "homepage": "https://github.com/nasa/cumulus/tree/master/tasks/post-to-cmr",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nasa/cumulus"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "ava",
+    "test-coverage": "nyc ava",
+    "build": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack",
+    "watch": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack --progress -w",
+    "prepublishOnly": "PRODUCTION=true npm run build"
```

Comment:
`prepublishOnly` [is deprecated](https://github.com/lerna/lerna/tree/master/commands/bootstrap#--ignore-prepublish), use `prepare` instead.